<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtGuiSimulator</class>
 <widget class="QMainWindow" name="QtGuiSimulator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>977</width>
    <height>715</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtGuiSimulator</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QPushButton" name="pushButton_play">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>39</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>play-button.jpg</normaloff>play-button.jpg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_parametrs">
        <property name="minimumSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>stop.png</normaloff>stop.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string>File</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLineEdit" name="linEdit_fileName">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtn_Save">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Save</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string>Program</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QComboBox" name="comboBox_program">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>3</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <item>
            <property name="text">
             <string>Img 0</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Img 1</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Img 2</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Img 3</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Img 4</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Img 5</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Img 6</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Img 7</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Img 8</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Img 9</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Img 10</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Img 11</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Img 12</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Img 13</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Img 14</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Img 15</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Img 16</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Img 17</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Img 18</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Img 19</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Img 20</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Img 21</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Img 22</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Img 23</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Img 24</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Img 25</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Img 26</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Img 27</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Img 28</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Img 29</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Img 30</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Img 31</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtn_Detail">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>1</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Detail</string>
           </property>
          </widget>
         </item>
        </layout>
        <zorder>pushButtn_Detail</zorder>
        <zorder>comboBox_program</zorder>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_ImgHist">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Image Hist</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_I_O_settng">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>I/O setting</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_advanc">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Advanced</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QtGuiDisplay" name="widget_DisplayImg" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Ignored" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>528</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="minimumSize">
         <size>
          <width>405</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>405</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="title">
         <string/>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Preferred</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>133</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <widget class="QPushButton" name="pushButton_SetupSimltr">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>Sensor Setup Simulator</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Program setting can be configured here by &lt;/p&gt;&lt;p&gt;following the step by step process&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_SensorSimltr">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>IV Sensor Simulator</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Operation result can be simulated by using imade&lt;/p&gt;&lt;p&gt;history&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Preferred</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>132</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
   <zorder></zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>977</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QtGuiDisplay</class>
   <extends>QWidget</extends>
   <header>qtguidisplay.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
